baldrick/config.py

```diff
@@ -9,20 +9,14 @@ def loads(text, tool='baldrick'):
     return Config(toml.loads(text)['tool'][tool])
 
 
-class Config:
+class Config(dict):
```

Comment:
`UserDict`?

Reply:
It looks like UserDict is now obsolete - https://legacy.python.org/dev/peps/pep-3108/

Reply:
(in Python 3.x)

Reply:
but its still here: https://docs.python.org/3/library/collections.html?highlight=collections#collections.UserDict :/

Reply:
also 3108 says "Useful bits moved to the 'collections' module."

Reply:
Having said that, I am fine with this the way it is.